build/checkstyle/tachyon_checks.xml

```diff
@@ -47,6 +47,11 @@
     <property name="fileExtensions" value=".java"/>
   </module>
   <module name="RegexpSingleline">
+    <property name="format" value="TODO[^\(]"/>
```

Comment:
It's not that simple. To achieve that, we have to use look-around in RE http://www.regular-expressions.info/lookaround.html.
But unfortunately, you cannot use regex inside a lookbehind, because they cannot apply a regular expression backwards.

I think the current rule should work in most cases, because it's hard to imagine a TODO followed by a parenthesis is not for adding author.